packages/web/src/pages/upload-page/fields/ReleaseDateField.tsx

```diff
@@ -0,0 +1,87 @@
+import { useCallback, useMemo } from 'react'
+
+import { IconCalendar } from '@audius/stems'
+import cn from 'classnames'
+import { useField } from 'formik'
+import moment from 'moment'
+
+import {
+  ContextualMenu,
+  SelectedValue
+} from 'components/data-entry/ContextualMenu'
+import { Text } from 'components/typography'
+
+import { EditFormValues } from '../components/EditPageNew'
+
+import { DatePickerField } from './DatePickerField'
+import styles from './ReleaseDateField.module.css'
+const messages = {
+  title: 'Release Date',
+  description:
+    'Specify a release date (in the past) for your music. Release date will affect the order of content on your profile and is visible to users.'
+}
+
+const RELEASE_DATE = 'releaseDate'
+
+export type ReleaseDateFormValues = {
+  [RELEASE_DATE]: moment.Moment
+}
+
+type ReleaseDateValue = EditFormValues[typeof RELEASE_DATE]
+
+export const ReleaseDateField = () => {
+  const [{ value }, , { setValue }] = useField<ReleaseDateValue>(RELEASE_DATE)
```

Comment:
you might want to merge in my multi-track changes before reimplementing the rest of the forms. I had to change a lot of the field, initialValues, and onSubmits

Reply:
yeah was thinking this when i saw the merge conflict, was like whew glad we paused here